sympy/vector/tests/test_implicitregion.py

```diff
@@ -36,12 +35,9 @@ def test_regular_point():
 
 def test_singular_points_and_multiplicty():
     r1 = ImplicitRegion((x, y, z), Eq(x + y + z, 0))
-    assert r1.singular_points() == FiniteSet((-y - z, y, z))
-    assert r1.multiplicity((0, 0, 0)) == 1
-    assert r1.multiplicity((-y - z, y, z)) == 1
+    assert r1.singular_points() == EmptySet
```

Comment:
What exactly is the underlying call to `nonlinsolve` here?

Reply:
The call to `nonlinsolve` is here:
https://github.com/sympy/sympy/blob/5f7edff136e834a287cb61238f855169011ba9b6/sympy/vector/implicitregion.py#L283-L304

Reply:
Yes, but in this particular case what are the exact inputs (`eq_list` and `self.variables`)? How would I demonstrate what has changed just by calling `nonlinsolve` directly.

Reply:
In this case, `eq_list` is `[x + y + z, 1, 1, 1]` and `list(self.variables)` is `[x, y, z]`. The system clearly has no solution, so `EmptySet` is the correct return value

Reply:
Oh, okay. It would be good to add the directly to the nonlinsolve tests.

Reply:
Same goes for anything else that was detected indirectly like this.

Reply:
Ok, I'll add the tests shortly.